algorithms/repository_rating.py

```diff
@@ -413,19 +417,19 @@ def calcMeritScore(self):
     def calcRepoScore(self):
         self.repoPoints = self.basePoints + self.activityPoints + self.inclusivityPoints + self.meritPoints
 
+"""
+RepositoryMetrics.token = "<token>"
 
-    
-accessToken = "03e5d817f468829fd9b3307f55de055461460c1a"
-headers = {"Authorization": "bearer "+ accessToken }
 
 username = input("Enter the owner name: ")
 reponame = input("Enter the repository name: ")
-r=RepositoryMetrics(str(reponame), str(username), accessToken)
+r=RepositoryMetrics(str(reponame), str(username), RepositoryMetrics.token)
 
 print()
 print("The Base Points: " + str(r.basePoints))
 print("The Activity Points: " + str(r.activityPoints))
 print("The Inclusivity Points: " + str(r.inclusivityPoints))
 print("The Merit Points: " + str(r.meritPoints))
 print()
-print("The Repository Score is " + str(r.repoPoints))
\ No newline at end of file
+print("The Repository Score is " + str(r.repoPoints))
```

Comment:
Comment these outputs as they wouldn't be required now as we have integrated it with Django.